smart_card_connector_app/src/application_unittest.cc

```diff
@@ -79,19 +75,82 @@ class SmartCardConnectorApplicationTest : public ::testing::Test {
         Value(Value::Type::kDictionary)));
   }
 
+  // Enables the specified fake USB devices.
+  void SetUsbDevices(
+      const std::vector<TestingSmartCardSimulation::Device>& devices) {
+    smart_card_simulation_.SetDevices(devices);
+  }
+
+  // Returns all "reader_init_add" messages that were sent to JS (they are sent
+  // when a new reader starts to be initialized).
+  const std::vector<std::string>& reader_init_add_messages() const {
+    return reader_init_add_messages_;
+  }
+  // Returns all "reader_finish_add" messages that were sent to JS (they are
+  // sent when a reader initialization finishes).
```

Comment:
optional nit: I think the comments would be better at the variables themselves, not at their getters